latex/main.tex

```diff
@@ -38,21 +55,76 @@ \section{The website and how it works}
 ~\\
```

Comment:
De handleiding is er om de gebruiker uit te leggen hoe alles werkt, om dan nog in de titel van een hoofdstuk te zeggen "and how it works" is erg dubbel.